### DISCUSSION TITLE
Sep 18 office hours

### DISCUSSION THREAD

**Discussion by dckc** [Agoric/agoric-sdk#10107] [Office Hours]
[zoom recording](https://agoric.zoom.us/rec/share/fI0fiZYJEgHJCIhbHik_vjmLuRdwyKwZW5MKICvyO78TIyfexG5uGIdM8H6rdYWY.vGW6hRDet8SBN9AH)
Passcode: `&8Fv22B*`

tab dump:

[test(a3p): migrate wallet related test to z:acceptance · bytepitch/agoric-sdk@6afaba3](https://github.com/bytepitch/agoric-sdk/commit/6afaba3c437c03d580ae29c1acaf7c5659f73d42#)
[Pull requests · Agoric/agoric-sdk](https://github.com/Agoric/agoric-sdk/pulls?q=is%3Apr+core-eval+eslint+is%3Aclosed)
[agoric-sdk/multichain-testing/tools/sleep.ts at f291362e4bc62ec31552cef29ec1a5f5cfbf7bd7 · Agoric/agoric-sdk](https://github.com/Agoric/agoric-sdk/blob/f291362e4bc62ec31552cef29ec1a5f5cfbf7bd7/multichain-testing/tools/sleep.ts#L62)
[Agoric - Deploy Secure DeFi in JavaScript](https://agoric.com/)
[Office Hours · Agoric/agoric-sdk Wiki](https://github.com/Agoric/agoric-sdk/wiki/Office-Hours)
[Latest topics - Agoric Community Forum](https://community.agoric.com/latest)
[Release agoric-upgrade-17-rc0 · Agoric/agoric-sdk](https://github.com/Agoric/agoric-sdk/releases/tag/agoric-upgrade-17-rc0)
[Blaming agoric-sdk/packages/builders/scripts/vats/init-orchestration.js at master · Agoric/agoric-sdk](https://github.com/Agoric/agoric-sdk/blame/master/packages/builders/scripts/vats/init-orchestration.js)
[orchestration - ICA Controller - Account - Create account on host chain · Issue #9064 · Agoric/agoric-sdk](https://github.com/Agoric/agoric-sdk/issues/9064)
[agoric-sdk/packages/orchestration/src/orchestration-api.ts at master · Agoric/agoric-sdk](https://github.com/Agoric/agoric-sdk/blob/master/packages/orchestration/src/orchestration-api.ts#L168)
[How to performance-test a contract? Available tools? · Agoric/agoric-sdk · Discussion #9941](https://github.com/Agoric/agoric-sdk/discussions/9941)

**Comment by tgrecojs** [comment]
hey @dckc can you share zoom recording when it's available? Thanks! 🙂

**Comment by dckc** [comment]
done: [zoom recording](https://agoric.zoom.us/rec/share/fI0fiZYJEgHJCIhbHik_vjmLuRdwyKwZW5MKICvyO78TIyfexG5uGIdM8H6rdYWY.vGW6hRDet8SBN9AH)
Passcode: `&8Fv22B*`
